### PR TITLE
service token example update

### PR DESCRIPTION
**Pull request by kennyj42**

### CODE REVIEW

**Review by jroyal — APPROVED** [review]

**Review by kennyj42 — COMMENTED** [review]
small tweak

File: content/cloudflare-one/identity/service-tokens.md [hunk 1]
```diff
@@ -37,7 +37,17 @@ To authenticate to an Access application using your service token, add the follo
 
 `CF-Access-Client-Secret: <Client Secret>`
 
-If the service token is valid, Access generates a JWT scoped to the application. All subsequent requests with that JWT will succeed until the expiration of that JWT.
+If the service token is valid, Access generates a JWT scoped to the application in the form of a cookie. All subsequent requests with that JWT will succeed until the expiration of that JWT. The JWT `CF_Authorization` should be set as a cookie on any subsequent requests.
```
Comment (ranbel) [inline content/cloudflare-one/identity/service-tokens.md]: Does this mean that only the first request to an application should include `cf-access-client-id` and `cf-access-client-secret`? Should we provide an example for how to add those headers?

Comment (kennyj42) [reply content/cloudflare-one/identity/service-tokens.md]: Yes, that's correct. Like just the example CURL? -h

Comment (ranbel) [reply content/cloudflare-one/identity/service-tokens.md]: yes, since we include examples for the subsequent requests

Comment (kennyj42) [reply content/cloudflare-one/identity/service-tokens.md:40]: curl -H "CF-Access-Client-Id: dfcafc3f102dd48b7d00256cfd940325.access" -H "CF-Access-Client-Secret: 82fd1c60ccf63440427bcff8d8c8c55d351930975e0b3a67c9b13bd3ef08ab56" https://cookieprint.kennyatx.com

Comment (kennyj42) [reply content/cloudflare-one/identity/service-tokens.md]: curl -H "CF-Access-Client-Id: <clientID>" -H "CF-Access-Client-Secret: <secret>" https://cookieprint.kennyatx.com

**Comment by ranbel** [comment]
@kennyj42 Does this latest version make sense?

**Review by kennyj42 — COMMENTED** [review]
Yep! let's ship :)

**Review by ranbel — APPROVED** [review]

**Pull request merged**